app/routes/project/posts/post.js

```diff
@@ -32,6 +32,16 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    save(post) {
+      post.save().catch((error) => {
```

Comment:
So here you are probably going to want to return the promise.
`return post.save().catch((err....`